daemons/pacemakerd/pcmkd_corosync.c

```diff
@@ -95,7 +95,10 @@ cluster_disconnect_cfg(void)
 {
     if (cfg_handle) {
 #ifdef HAVE_COROSYNC_CFG_TRACKSTART
-        corosync_cfg_trackstop(cfg_handle);
+        /* Ideally, we would call corosync_cfg_trackstop(cfg_handle) here, but a
```

Comment:
Doesn't it hang in pcmkd_shutdown_corosync as well?
With 2 places it might make sense to have HAVE_COROSYNC_CFG_TRACKSTOP and move the logic (simplest case never on ;-) ) to configure.ac as not to forget about it when it is working again.

Reply:
Ouch, I missed that too :-/

I'll just functionize the trackstop+finalize to have the comment in one place. It's OK if we forget it since it doesn't have any real effect at exit :)